<launch>
<arg name="drone_id" default="1"/>
<node pkg = "trajectory_follower" name = "trajectory_follower_node" type = "trajectory_follower_node" output="screen" ns="drone_$(arg drone_id)" >
    <rosparam file="$(find trajectory_follower)/param/param.yml" command="load"/>
    <param name="drone_id" value="$(arg drone_id)"/>
</node>
</launch>
